data/common.yaml

```diff
@@ -1 +1,18 @@
---- {}
+---
+vcsrepo::manage::bzr::package_name: bzr
```

Comment:
please put the data directly into the classes. Puppet-strings will pick it up and document it in the README.md

Reply:
Can do, wasn't sure if various platforms would need an override... I know nothing about the BSD world.